### ISSUE TITLE
Accesibility is missing

### ISSUE THREAD

**Issue by facupascale** [ThakurBallary/react-native-radio-buttons-group#43]
I want to used it for my own library components but it doesnt have any accessibility prop and its a requeriment in the proyect Im working on, its something easy to add and it would not take more than a few lines. I think its a good improvement to have it

**Comment by aviralshastri** [comment]
How to change colour of the text?

**Comment by despin** [comment]
@facupascale What props do you need for your accessibility thing to be working? I have (I think) the same issue as you. I am writing some tests for my app, and cannot trigger `onPress` on some item because I can't find it with `getByTestId`.

**Comment by carlaTatiana** [comment]
Hi! Same situation here. It doesn't have any accessibility prop

**Comment by ThakurBallary** [comment]
Added `accessibilityLabel` prop to radio button in [v3.0.3 Accessibility](https://github.com/ThakurBallary/react-native-radio-buttons-group/releases/tag/v3.0.3)
